candies/cli/parsers/parser.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABCMeta, abstractmethod
+from typing import List, Dict, Any
+
+
+class Parser(metaclass=ABCMeta):
+    """A parser for command line arguments."""
+
+    @abstractmethod
+    def parse(self, args: List[str]) -> Dict[str, Any]:
+        """Parses the specified arguments."""
```

Comment:
Shouldn't we document here the format of arguments and returned value? Or is it too much for the abstract classes?

Reply:
We definitely should. I'll update the documentation.